config/locales/en.yml

```diff
@@ -39,6 +39,15 @@ en:
     posts: Posts
     remote_follow: Remote follow
     unfollow: Unfollow
+  activitypub:
```

Comment:
For reference, here's the AS2 section on natural language values: https://www.w3.org/TR/activitystreams-core/#naturalLanguageValues

It looks like we can either just do English or provide all of the language options that Mastodon supports in the same JSON object. I'd err on the side of the former for now, but whether to add full language support later is your call.